assembly/compliance-tool/src/main/java/org/eclipse/sw360/antenna/frontend/compliancetool/main/AntennaComplianceToolOptions.java

```diff
@@ -151,13 +189,16 @@ static String helpMessage() {
     private static Optional<String> getComplianceModeFromSwitches(Set<String> switches) {
         boolean updater = hasSwitch(switches, SWITCH_UPDATER_SHORT) || hasSwitch(switches, SWITCH_UPDATER_LONG);
         boolean exporter = hasSwitch(switches, SWITCH_EXPORTER_SHORT) || hasSwitch(switches, SWITCH_EXPORTER_LONG);
+        boolean reporter = hasSwitch(switches, SWITCH_REPORTER);
 
         if (updater && exporter) {
```

Comment:
The checking logic is now more complex; the new switch has to be taken into account as well.